ckanext/stadtzh_losdharvest/profiles.py

```diff
@@ -20,17 +20,17 @@
 log = logging.getLogger(__name__)
 
 BASE = Namespace("https://ld.stadt-zuerich.ch/schema/")
-BASEINT = Namespace("https://ld.integ.stadt-zuerich.ch/schema/")
+BASEINT = Namespace("https://ld.integ.stzh.ch/schema/")
 QUDT = Namespace("http://qudt.org/schema/qudt#")
 VOID = Namespace("http://rdfs.org/ns/void#")
 OWL = Namespace("http://www.w3.org/2002/07/owl#")
 XSD = Namespace("http://www.w3.org/2001/XMLSchema#")
-DCTERMS = Namespace("http://purl.org/dc/terms/")
+DCTERMS = Namespace("https://purl.org/dc/terms/")
```

Comment:
This should not be changed: the namespace as specified in https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#section-1 is `http://purl.org/dc/terms/`. Accessing that url in a browser redirects to https://www.dublincore.org/specifications/dublin-core/dcmi-terms/, so there is no security gain by changing to use `https` here.

Reply:
reverted